docs/marshal.rst

```diff
@@ -13,24 +13,24 @@ Well-known types
 
 The following types are currently handled by Proto Plus:
 
-=================================== ======================= ========
-Protocol buffer type                Python type             Nullable
-=================================== ======================= ========
-``google.protobuf.BoolValue``       ``bool``                     Yes
-``google.protobuf.BytesValue``      ``bytes``                    Yes
-``google.protobuf.DoubleValue``     ``float``                    Yes
-``google.protobuf.Duration``        ``datetime.timedelta``         –
-``google.protobuf.FloatValue``      ``float``                    Yes
-``google.protobuf.Int32Value``      ``int``                      Yes
-``google.protobuf.Int64Value``      ``int``                      Yes
-``google.protobuf.ListValue``       ``MutableSequence``            –
-``google.protobuf.StringValue``     ``str``                      Yes
-``google.protobuf.Struct``          ``MutableMapping``             –
-``google.protobuf.Timestamp``       ``datetime.datetime``        Yes
-``google.protobuf.UInt32Value``     ``int``                      Yes
-``google.protobuf.UInt64Value``     ``int``                      Yes
-``google.protobuf.Value``           JSON-encodable values        Yes
-=================================== ======================= ========
+=================================== ================================================== =========
+Protocol buffer type                Python type                                        Nullable
+=================================== ================================================== =========
+``google.protobuf.BoolValue``       ``bool``                                           Yes
+``google.protobuf.BytesValue``      ``bytes``                                          Yes
+``google.protobuf.DoubleValue``     ``float``                                          Yes
+``google.protobuf.Duration``        ``datetime.timedelta``                               –
+``google.protobuf.FloatValue``      ``float``                                          Yes
+``google.protobuf.Int32Value``      ``int``                                            Yes
+``google.protobuf.Int64Value``      ``int``                                            Yes
+``google.protobuf.ListValue``       ``MutableSequence``                                  –
+``google.protobuf.StringValue``     ``str``                                            Yes
+``google.protobuf.Struct``          ``MutableMapping``                                  –
+``google.protobuf.Timestamp``       ``proto.datetime_helpers.DatetimeWithNanoseconds`` Yes
```

Comment:
This arguably obscures more than it elucidates. It might be more valuable to put `datetime.datetime` with a footnote that says, "Actually, a subclass with nanosecond precision". This is consistent with the use of `MutableSequence` and `MutableMapping`, which are actually interfaces.

What 99% of people want to know is that they can use it like a datetime.